Sources/TSCBasic/Process.swift

```diff
@@ -366,7 +366,10 @@ public final class Process: ObjectIdentifierProtocol {
         defer { posix_spawn_file_actions_destroy(&fileActions) }
 
         // Workaround for https://sourceware.org/git/gitweb.cgi?p=glibc.git;h=89e435f3559c53084498e9baad22172b64429362
-        let devNull = strdup("/dev/null")
+        // Change allowing for newer version of glibc
+        guard let devNull = strdup("/dev/null") as Optional else {
```

Comment:
What is the return type for `strdup` on new glibc?

Reply:
From a quick overview of the changelog, i can't seem to find the change, will quickly do a look through the sourcecode and git history

Reply:
You can do `print(strdup.self)` to print the type that is imported in Swift.

Reply:
```swift
import Glibc
print(strdup.self)
```
Just gives me:
```
(Function)
```

Reply:
What about print(type(of: strdup))

Reply:
@aciidb0mb3r that gives me:
```
(UnsafePointer<Int8>) -> Optional<UnsafeMutablePointer<Int8>>
```

Reply:
Ah. Then I think `strdup("/dev/null")!` should work here as well.

Reply:
or you can just drop `as Optional` if you want to keep the guard around. IDK if this can actually fail.

Reply:
@aciidb0mb3r okay i have removed `as Optional`, but kept the guard, if you feel strongly against it, i will remove it .. But in my opinion better safe than sorry :D

Reply:
Cool, that sounds good to me. I agree that we shouldn't force unwrap but not all C APIs are imported correctly in Swift. Anyway, thanks for working on this!